### PR TITLE
Remove SlackTextViewController

### LINKED ISSUE #1319: Remove SlackTextViewController
As of #1307 we no longer use this pod, so we should remove it from the project both reference wise but also the directory in `Local Pods/`

https://github.com/rnystrom/GitHawk/blob/master/Podfile#L31

### PR DESCRIPTION
**Pull request by vanyaland**
#### Summary:
- Remove SlackTextViewController pod
- Replace SLKTextView with plain UITextView
https://github.com/rnystrom/GitHawk/blob/master/Classes/Image%20Upload/ImageUploadTableViewController.swift#L26

Closes #1319.

Thanks!

### CODE REVIEW

File: Classes/Image Upload/ImageUploadTableViewController.swift [hunk 1]
```diff
@@ -235,3 +232,53 @@ extension ImageUploadTableViewController: UITextFieldDelegate {
...
+        
+        return true
+    }
+    
+    func textViewDidChangeSelection(_ textView: UITextView) {
+        if self.view.window != nil && textView.textColor == Styles.Colors.Gray.light.color {
+            setCursorToStart(textView: textView)
+        }
+    }
+}
```
Comment (rnystrom) [inline Classes/Image Upload/ImageUploadTableViewController.swift]: Hmm I think we should maybe just move the placeholder `UILabel` from `MessageView` into `MessageTextView` and reuse that here? Would be a lot simpler. I can foresee some bugs from accidentally sending the placeholder text if actually its part of the `UITextView`.

**Comment by rnystrom** [comment]
So great! This will probably help our binary size. Left a comment about how we're doing placeholders, I think once we figure that out for the upload flow we'll be good!

File: Classes/Image Upload/ImageUploadTableViewController.swift (hunk 1 above)
Comment (vanyaland) [reply Classes/Image Upload/ImageUploadTableViewController.swift]: Totally agree. I will make appropriate changes.

**Comment by vanyaland** [comment]
Hey @rnystrom , please review.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Fantastic. Going to hold on merging this til 1.17.0 ships and then begin the great local-pod migration.

**Comment by BasThomas** [comment]
This crashes on `master`:

```
'MessageViewController.MessageAutocompleteController' does not implement methodSignatureForSelector: -- trouble ahead
Unrecognized selector -[MessageViewController.MessageAutocompleteController containsValueForKey:]
2018-01-20 20:54:43.321137+0100 Freetime[82749:20628764] Unrecognized selector -[MessageViewController.MessageAutocompleteController containsValueForKey:]
```

**Comment by rnystrom** [comment]
@vanyaland 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by vanyaland** [comment]
@BasThomas @rnystrom Couldn't reproduce.
As of this PR I doesn't affect MessageAutocompleteController behavior.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by BasThomas** [comment]
Oh, hah - seems to have been something in Derived Data. Great. Sorry @vanyaland!

**Comment by rnystrom** [comment]
Thanks so much for taking a look @vanyaland!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**